wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Data.kt

```diff
@@ -46,6 +46,16 @@ data class Payment(
   @SerialName("id_type") val idType: String
 )
 
+@Serializable
+data class FeeDetails(
+  val total: String,
+  val asset: String? = null,
```

Comment:
Asset should be non-null 
Except that LGTM

Reply:
Hmm just curious why `amountFeeAsset` can be null? I was thinking about matching the nullability